gandi/cli/tests/commands/test_config.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+""" Configuration namespace tests. """
+
+import os
+
+from .base import CommandTestCase
+from gandi.cli.commands import config
+
+
+class ConfigTestCase(CommandTestCase):
+
+    def test_get_empty(self):
+        result = self.invoke_with_exceptions(config.get, [])
+        self.assertEqual(result.exit_code, 2)
+
+    def test_get_unknown(self):
+        result = self.invoke_with_exceptions(config.get, ['unknown-key'])
+        self.assertEqual(result.output, """No value found.
```

Comment:
You should be consistent and keep the same notation as in other tests, so it doesn't disturb people looking in the code the first time.